notebooks/04_model_select_and_optimize/nni_surprise_svd.ipynb

```diff
@@ -32,17 +32,19 @@
   },
```

Comment:
<p>there is an error here, could you please check?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Hi Miguel, what environment did the error occur on? I made the modification to add the <code>ast.literal_eval</code> while I was upgrading to nni v1.5 as it was outputting an error before hand. This currently works on a Azure DSVM with the reco_gpu conda environment, so I'll need to take a closer look at the environment this occurred on </p><p> </p>

Reply:
this is the error I see on github:
```
ValueError Traceback (most recent call last)<ipython-input-71-c9381fa3d2ee> in <module>
1 check_metrics_written(wait=WAITING_TIME, max_retries=MAX_RETRIES)
----> 2 trials, best_metrics, best_params, best_trial_path = get_trials('maximize')
~/projects/recommenders/reco_utils/tuning/nni/nni_utils.py in get_trials(optimize_mode)
111 trials = [
112 (ast.literal_eval(eval(trial["finalMetricData"][0]["data"])), trial["logPath"].split(":")[-1])
--> 113 for trial in all_trials
114 ]
115 sorted_trials = sorted(
~/projects/recommenders/reco_utils/tuning/nni/nni_utils.py in <listcomp>(.0)
111 trials = [
112 (ast.literal_eval(eval(trial["finalMetricData"][0]["data"])), trial["logPath"].split(":")[-1])
--> 113 for trial in all_trials
114 ]
115 sorted_trials = sorted(
/anaconda/envs/recommenders_gpu/lib/python3.6/ast.py in literal_eval(node_or_string)
83 return left - right
84 raise ValueError('malformed node or string: ' + repr(node))
---> 85 return _convert(node_or_string)
86
87
/anaconda/envs/recommenders_gpu/lib/python3.6/ast.py in _convert(node)
82 else:
83 return left - right
---> 84 raise ValueError('malformed node or string: ' + repr(node))
85 return _convert(node_or_string)
86
ValueError: malformed node or string: {'rmse': 1.160248985195997, 'default': 0, 'ndcg_at_k': 0}
```